Release 1.0.0

wonderlandpark opened the pull request:
close #9
close #10
close #11
close #12
close #16
close #18
close #19
close #34
close #35
close #36

SaidBySolo committed:
style: edit typo

SaidBySolo committed:
chore: not using env

SaidBySolo committed:
fix: apply request changes

SaidBySolo committed:
chore: add env

SaidBySolo committed:
Merge pull request #13 from SaidBySolo/develop

feat: meal info

SaidBySolo committed:
chore: add build config

SaidBySolo committed:
refactor: meal little optimization,  feat: academic_calendar get schoolinfo

SaidBySolo committed:
feat: academic_calendar complete

SaidBySolo committed:
Merge pull request #1 from SaidBySolo/feature

feat: academic_calendar

SaidBySolo committed:
style: rename filename

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
style: rename func name

SaidBySolo committed:
style: rename filename

SaidBySolo committed:
fix: check channel

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
Merge pull request #15 from SaidBySolo/develop

refactor: if statement, feat: academic_schedule

kndlr committed:
feat: add iamschool

kndlr committed:
Revert "chore: add build config"

This reverts commit e00e7f1edc978b04bda198dd0fa4d22b321126df.

SaidBySolo committed:
Merge pull request #20 from callistoteam/staging

deploy beta

kndlr committed:
feat: iamschool article

kndlr committed:
code refactor

kndlr committed:
pull upstream

kndlr committed:
style: apply code style

kndlr committed:
style(iamschool): apply code style

kndlr committed:
fix: apply request changes

SaidBySolo committed:
feat: install private package

kndlr committed:
Merge pull request #1 from callistoteam/SaidBySolo-patch1

feat: install private package

kndlr committed:
Merge branch 'SaidBySolo-patch1' of https://github.com/callistoteam/schoolbot into develop

kndlr committed:
style: apply code style

kndlr committed:
Merge branch 'develop' of https://github.com/COIN-KR/schoolbot into develop

SaidBySolo committed:
Merge pull request #21 from COIN-KR/develop

feat: iamschool article

SaidBySolo committed:
chore: add run.sh

SaidBySolo committed:
Merge pull request #23 from callistoteam/SaidBySolo-patch1

chore: add run.sh

SaidBySolo committed:
Merge pull request #22 from callistoteam/staging

deploy beta

SaidBySolo committed:
chore: edit build config

SaidBySolo committed:
fix: dir

SaidBySolo committed:
fix: permisson denied

SaidBySolo committed:
fix: permisson denied

SaidBySolo committed:
fix: run.sh

SaidBySolo committed:
chore: shell script not need in this branch

SaidBySolo committed:
Merge branch 'staging' into SaidBySolo-patch1

SaidBySolo committed:
Merge pull request #24 from callistoteam/SaidBySolo-patch1

chore: shell script not need in this branch

SaidBySolo committed:
Merge branch 'beta' into staging

SaidBySolo committed:
chore: this branch not need build config

SaidBySolo committed:
feat: timetable

SaidBySolo committed:
Merge branch 'staging' into develop

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
chore: install private dependency

SaidBySolo committed:
Merge pull request #26 from SaidBySolo/develop

feat: timetable

SaidBySolo committed:
feat: help, fix: if info 200 raise unexpected error

SaidBySolo committed:
Merge pull request #1 from SaidBySolo/feature

feat: help, fix: if info 200 raise unexpected error

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
chore: edit install private package

SaidBySolo committed:
chore: edit typo

SaidBySolo committed:
chore: edit typo

SaidBySolo committed:
chore: edit build setup

SaidBySolo committed:
Merge branch 'master' into staging

SaidBySolo committed:
Merge branch 'staging' into develop

SaidBySolo committed:
style: edit typo

SaidBySolo committed:
Merge pull request #27 from SaidBySolo/develop

feat: help, fix: if info 200 raise unexpected error

SaidBySolo committed:
feat: add general cogs

SaidBySolo committed:
style: change version name

SaidBySolo committed:
style: change presence sentence

SaidBySolo committed:
Merge pull request #30 from callistoteam/SaidBySolo-patch-1

Ready to release 1.0.0

SaidBySolo committed:
style: edit typo

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
feat: add timetable command

SaidBySolo committed:
fix: apikey error

kndlr committed:
feat: setting, search, db

SaidBySolo committed:
feat: change code owner

SaidBySolo committed:
Merge remote-tracking branch 'upstream/staging' into develop

SaidBySolo committed:
style: apply request change

SaidBySolo committed:
refactor: apply request changes

kndlr committed:
fix: search strftime

SaidBySolo committed:
refactor: apply request changes

SaidBySolo committed:
refactor: apply request changes

SaidBySolo committed:
style: delete remark

SaidBySolo committed:
refactor: check several schools

SaidBySolo committed:
refactor: apply chnage requests

fix: timezone issue
feat: debug module

SaidBySolo committed:
Merge pull request #38 from SaidBySolo/develop

refactor: apply request changes

SaidBySolo committed:
chore: add dependency

chore: some change build config

SaidBySolo committed:
chore: add env

SaidBySolo committed:
Merge pull request #39 from SaidBySolo/develop

chore: add dependency

SaidBySolo committed:
chore: add branch

SaidBySolo committed:
Merge pull request #40 from SaidBySolo/develop

chore: add branch

SaidBySolo committed:
style: change prefix, delete not using code

SaidBySolo committed:
fix: str - int error

SaidBySolo committed:
feat: check owner, add limit traceback

SaidBySolo committed:
Merge pull request #42 from SaidBySolo/develop

fix: str - int error, feat: handle error, limit traceback log, style: edit help

kndlr committed:
feat: db meal, schedule

SaidBySolo committed:
style: embed styling

SaidBySolo committed:
Merge pull request #44 from SaidBySolo/develop

style: embed styling

SaidBySolo committed:
fix: cant send embed

SaidBySolo committed:
Merge pull request #46 from SaidBySolo/develop

fix: cant send embed

kndlr committed:
feat: db timetable

kijk2869 committed:
style: dependent version management

kijk2869 committed:
style: apply dependabot

kijk2869 committed:
Merge pull request #48 from callistoteam/kijk2869-patch-1

style: dependent version management

kijk2869 committed:
docs: add contact

kndlr committed:
style: apply code style

kndlr committed:
fix: help command

kndlr committed:
fix: apply request changes

wonderlandpark committed:
feat: #36

kndlr committed:
fix: apply request changes

kndlr committed:
fix: apply request changes

wonderlandpark committed:
fix: request querystring

wonderlandpark committed:
Merge branch 'staging' into i'm-so-sexy

wonderlandpark committed:
fix: hard coding

Co-authored-by: 매리 <mareowst@gmail.com>

kndlr committed:
fix: apply request changes

kndlr committed:
fix: apply request changes

wonderlandpark committed:
fix: hard coding

Co-authored-by: Ryu JuHeon <saidbysolo@gmail.com>

wonderlandpark committed:
fix: encode uri

wonderlandpark committed:
fix: hard coding

wonderlandpark committed:
feat: os env with schoolbot api token

wonderlandpark committed:
Merge branch 'feature-meal-prerendering' into i'm-so-sexy

wonderlandpark committed:
Merge pull request #54 from callistoteam/i'm-so-sexy

I'm so sexy

wonderlandpark committed:
feat: urllib parse quote

wonderlandpark committed:
Merge branch 'feature-meal-prerendering' of https://github.com/callistoteam/schoolbot into feature-meal-prerendering

wonderlandpark committed:
fix: env name

wonderlandpark committed:
feat: format as black

wonderlandpark committed:
Merge pull request #53 from callistoteam/feature-meal-prerendering

feat: added meal prerendering ( #36 )

wonderlandpark committed:
Merge branch 'staging' into develop

kndlr committed:
fix: apply request changes

kndlr committed:
fix: apply request changes

kndlr committed:
fix: apply request changes

SaidBySolo committed:
Merge pull request #49 from COIN-KR/develop

feat: aiomysql

wonderlandpark committed:
Update python-app-lint.yml

wonderlandpark committed:
Update python-app-test.yml

Yoruroong committed:
Update README.md

Yoruroong committed:
Update README.md

Yoruroong committed:
fix: apply request changes

fossabot committed:
Add license scan report and status

Signed off by: fossabot <badges@fossa.com>

kijk2869 committed:
Merge pull request #57 from fossabot/add-license-scan-badge

Add license scan report and status

SaidBySolo committed:
fix: question mark is the most used, so it can be inconvenient.

SaidBySolo committed:
style: delete remark

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve converstion

SaidBySolo committed:
style: dupe func name

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
Merge pull request #1 from SaidBySolo/feature

Resolve conversation and fix some problem

SaidBySolo committed:
refactor: unused module

SaidBySolo committed:
style: typo

SaidBySolo committed:
refactor: neispy 3.0 update

SaidBySolo committed:
refactor: neispy update 3.0

SaidBySolo committed:
Merge pull request #2 from SaidBySolo/feature

refactor: neispy update 3.0

SaidBySolo committed:
style: typo

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
chore: check lint

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
Merge pull request #58 from SaidBySolo/develop

neispy 3.0 update corver, resolve conversation

dependabot[bot] committed:
chore(deps): bump neispy from 2.0.7 to 3.0.0

Bumps [neispy](https://github.com/SaidBySolo/neispy) from 2.0.7 to 3.0.0.
- [Release notes](https://github.com/SaidBySolo/neispy/releases)
- [Commits](https://github.com/SaidBySolo/neispy/commits)

Signed-off-by: dependabot[bot] <support@github.com>

kijk2869 committed:
Merge pull request #60 from callistoteam/dependabot/pip/neispy-3.0.0

chore(deps): bump neispy from 2.0.7 to 3.0.0

kijk2869 committed:
Update README.md

kijk2869 committed:
fix https://github.com/callistoteam/schoolbot/issues/62

kijk2869 committed:
Create python-type-hint.yml

kijk2869 committed:
Update python-type-hint.yml

kijk2869 committed:
Delete python-type-hint.yml

kijk2869 committed:
일해라 라솔

kndlr committed:
docs: default value

wonderlandpark committed:
Merge branch 'production' into staging

kijk2869 committed:
add: sql query for db

SaidBySolo committed:
refactor: migration

SaidBySolo committed:
fix: list out of range

SaidBySolo committed:
feat: use dictcursor

SaidBySolo committed:
refactor: dict cursor

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
Merge pull request #67 from SaidBySolo/fix

Maybe fixed db problem

SaidBySolo committed:
fix: receiving wrong argument value

SaidBySolo committed:
Merge branch 'staging' into fix

SaidBySolo committed:
Merge pull request #69 from SaidBySolo/fix

fix: receiving wrong argument value

SaidBySolo committed:
fix: receiving wrong argument value

SaidBySolo committed:
Merge branch 'fix' of https://github.com/SaidBySolo/schoolbot into fix

SaidBySolo committed:
Merge branch 'staging' into fix

SaidBySolo committed:
Merge pull request #71 from SaidBySolo/fix

I Believe fix timetable argument error

SaidBySolo committed:
fix: human error

SaidBySolo committed:
Merge branch 'fix' of https://github.com/SaidBySolo/schoolbot into fix

SaidBySolo committed:
Merge branch 'staging' into fix

kijk2869 committed:
Merge pull request #73 from SaidBySolo/fix

fix: human error

SaidBySolo committed:
fix: embed

SaidBySolo committed:
fix: typo and image

SaidBySolo committed:
Merge branch 'staging' into fix

SaidBySolo committed:
Merge pull request #77 from SaidBySolo/fix

fix: typo and add image

SaidBySolo committed:
fix: migration error

SaidBySolo committed:
fix: add_field

SaidBySolo committed:
fix: attribute error

dependabot[bot] committed:
chore(deps): bump neispy from 3.0.0 to 3.1.0

Bumps [neispy](https://github.com/SaidBySolo/neispy) from 3.0.0 to 3.1.0.
- [Release notes](https://github.com/SaidBySolo/neispy/releases)
- [Commits](https://github.com/SaidBySolo/neispy/commits)

Signed-off-by: dependabot[bot] <support@github.com>

SaidBySolo committed:
Merge pull request #81 from callistoteam/dependabot/pip/neispy-3.1.0

chore(deps): bump neispy from 3.0.0 to 3.1.0

SaidBySolo commented:
@kijk2869 @COIN-KR @SaidBySolo 

마지막 검토 예정

SaidBySolo reviewed (dismissed):
LGFM

kijk2869 commented on schoolbot/cogs/general/search.py:
```diff
@@ -0,0 +1,105 @@
+        else:
+            raise ValueError
+
+        embed = discord.Embed(title="검색 정보", description="")
```
```suggestion
        embed = discord.Embed(title="검색 정보")
```

kijk2869 commented on schoolbot/cogs/general/setting.py:
```diff
@@ -0,0 +1,74 @@
+import discord
+from discord.ext import commands
+
```
```suggestion
import re

```

kijk2869 commented on schoolbot/cogs/general/setting.py:
```diff
@@ -0,0 +1,74 @@
+    @commands.command(name="설정")
+    async def _setting(self, ctx, key: str = None, *, value: str = None):
+        if key and value:
+            args = " ".join(value.split("|")).split()
```
```suggestion
            args = re.split(r"\||\s", value)
```

SaidBySolo committed:
fix attribute

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,183 @@
+                if scarticles[position].images:
+                    embed.set_image(url=scarticles[position].images[0])
+                await msg.edit(embed=embed)
+                await msg.add_reaction("◀")
+                await msg.add_reaction("⏹")
+                await msg.add_reaction("▶")
```
```suggestion
                Bot.loop.create_task(asyncio.wait([msg.add_reaction(emoji) for emoji in ["◀", "⏹", "▶"]]))
```

SaidBySolo replied:
self.bot?

kijk2869 replied:
My mistake, Bot -> self.bot

kijk2869 commented on schoolbot/db.py:
```diff
@@ -0,0 +1,51 @@
+            return await cur.fetchone()
+
+
+async def create_user_data(id, neis_ae, neis_se, grade, class_nm, scclass):
+    async with pool.acquire() as conn:
+        async with conn.cursor() as cur:
+            await cur.execute(
+                "INSERT INTO `users`(`id`, `neis_ae`, `neis_se`, `grade`, `class_nm`, `class`) VALUES (%s,%s,%s,%s,%s,%s);",
+                (id, neis_ae, neis_se, grade, class_nm, scclass),
+            )
+            return
```
```suggestion
async def create_user_data(**kwargs):
    async with pool.acquire() as conn:
        async with conn.cursor() as cur:
            await cur.execute(
                "INSERT INTO `users`(`id`, `neis_ae`, `neis_se`, `grade`, `class_nm`, `class`) VALUES (%(id)s,%(neis_ae)s,%(neis_se)s,%(grade)s,%(class_nm)s,%(scclass)s);",
                kwargs,
            )
            return
```

kijk2869 commented on schoolbot/db.py:
```diff
@@ -0,0 +1,51 @@
+            return
+
+
+async def update_school(id, neis_ae, neis_se, grade, class_nm, scclass):
+    async with pool.acquire() as conn:
+        async with conn.cursor() as cur:
+            await cur.execute(
+                "UPDATE `users` SET `neis_ae` = %s, `neis_se` = %s, `grade` = %s, `class_nm` = %s, `class` = %s WHERE ID = %s;",
+                (neis_ae, neis_se, grade, class_nm, scclass, id),
+            )
+            return
```
```suggestion
async def update_school(**kwargs):
    async with pool.acquire() as conn:
        async with conn.cursor() as cur:
            await cur.execute(
                "UPDATE `users` SET `neis_ae` = %(neis_ae)s, `neis_se` = %(neis_se)s, `grade` = %(grade)s, `class_nm` = %(class_nm)s, `class` = %(class)s WHERE ID = %(id)s;",
                kwargs,
            )
            return
```

kijk2869 commented on utils/kst.py:
```diff
@@ -0,0 +1,10 @@
+import datetime
+
+
+def korean_date():
+    KST = datetime.timezone(datetime.timedelta(hours=9))
+    return datetime.datetime.now(tz=KST)
```
```suggestion
KST = datetime.timezone(datetime.timedelta(hours=9))

def korean_date():
    return datetime.datetime.now(tz=KST)
```

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
style: rename func

SaidBySolo committed:
hotfix: list out of range

kijk2869 committed:
hotfix: emoji sorting

kijk2869 committed:
hotfix: missing DB Query

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
hotfix: notice beta testing

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
hotfix: index out of range in setting

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
refactor: rm -rf / and some structure

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
feat: tortoise user model

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
feat: meal command

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
feat: meal command

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
feat: timetable command

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
add: docstrings

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
feat: academic schedule

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
fix: remove debug code

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
feat: search command

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
feat: article command

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
delete: sql query

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
remove database structure on readme

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
feat: setting command

Co-Authored-By: Ryu JuHeon <saidbysolo@gmail.com>

kijk2869 committed:
style: cleanup developement environment

kijk2869 committed:
style: format with isort

dependabot[bot] committed:
chore(deps): bump neispy from 3.1.0 to 3.2.0

Bumps [neispy](https://github.com/SaidBySolo/neispy) from 3.1.0 to 3.2.0.
- [Release notes](https://github.com/SaidBySolo/neispy/releases)
- [Commits](https://github.com/SaidBySolo/neispy/commits)

Signed-off-by: dependabot[bot] <support@github.com>

kijk2869 commented:
close #56 
close #75

kijk2869 committed:
Merge pull request #86 from callistoteam/dependabot/pip/neispy-3.2.0

chore(deps): bump neispy from 3.1.0 to 3.2.0

kijk2869 reviewed (dismissed):
내가 한건데 시발 어프루브지

kijk2869 committed:
refactor: utils extension

kijk2869 committed:
feat: auto mobile text & close #89

kijk2869 committed:
Merge branch 'staging' of https://github.com/callistoteam/schoolbot into staging

kijk2869 committed:
Update embed_to_text.py

kijk2869 committed:
feat what the blank field?

kijk2869 committed:
typo: markdown mistake

kijk2869 committed:
remove: print code

kijk2869 committed:
fix: format string typo

kijk2869 committed:
fix: User __repr__

kijk2869 committed:
Update embed_to_text.py

SaidBySolo committed:
feat: tos, privacy

SaidBySolo committed:
fix: typo

살다보면 실수할수도있는거지뭐

SaidBySolo committed:
fix: typo

진짜진짜 짜장면은 진짜진짜 진이먹네

SaidBySolo committed:
fix: for statement

SaidBySolo committed:
fix: for statement

SaidBySolo committed:
fix: position

SaidBySolo committed:
feat: regex

SaidBySolo committed:
style: black

kijk2869 committed:
feat: auto embed in edit

kijk2869 committed:
hotfix: missing Bot

kijk2869 committed:
hotfix: escape embed url

kijk2869 committed:
hotfix: description markdown

kijk2869 committed:
feat: autoembed image

kijk2869 committed:
feat: autoembed mobile

kijk2869 committed:
fix: delete mobile kwargs

kijk2869 committed:
add: article command aliases

kijk2869 committed:
fix: some typos

kijk2869 committed:
fix: version up

kijk2869 committed:
style: format with black and isort

kijk2869 committed:
style: format with isort

kijk2869 committed:
fix: use escape sequence

kijk2869 committed:
fix: duplicated alias

kijk2869 commented:
나 이거 걍 머지한다? 이제 1.0.0이 아닌듯

SaidBySolo commented:
force merge
